src/context/UserContext.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  createContext,
+  useContext,
+  useReducer,
+  ReactNode,
+  Dispatch,
+} from 'react';
+
+enum Currency {
+  USD = 'USD',
+  IDR = 'IDR',
+  JPY = 'JPY',
+  RUB = 'RUB',
+  EUR = 'EUR',
+  WON = 'WON',
+}
+
+enum Language {
```

Comment:
Move this enum to a separate file

Reply:
I moved Language enum to types folder and import to UserContect file.